src/core/App/App.js

```diff
@@ -1,19 +1,32 @@
-import { ThemeProvider } from "styled-components";
-import MovieBrowser from "../../feature/MovieBrowser";
-import { GlobalStyle } from "./GlobalStyles";
-import { theme } from "./theme";
-import { Pagination } from "./Pagination/index";
+import { HashRouter, Redirect, Route, Switch } from "react-router-dom";
+
+
 
 
 function App() {
   return (
-    <ThemeProvider theme={theme}>
-      <GlobalStyle />
-      <MovieBrowser />
+    <HashRouter>
+      <Header />
+      <Switch>
+        <Route path={toMovie()}>
+          <MoviePage />
+        </Route>
+        <Route path={toMovies()}>
+          <MoviesList />
+        </Route>
+        <Route path={toPerson()}>
+          <ProfilePersonPage />
+        </Route>
+        <Route path={toPeople()}>
+          <PeopleList />
+        </Route>
+        <Route>
+          <Redirect to={toMovies()} />
+        </Route>
+      </Switch>
```

Comment:
I think we should make new file for this part  and import it to APP.

Reply:
You're right, in the afternoon at the meeting we will determine the appearance of App.js and make changes and remove the conflict in merge